src/job/controller.ts

```diff
@@ -128,4 +132,40 @@ async function cancelJob(_req: express.Request, res: express.Response) {
   );
 }
 
+async function setStdLotSize(req: express.Request, res: express.Response) {
+  ExpressHandler(
+    async () => {
+      const { jobId, lotSize } = req.body;
+
+      console.error(jobId, lotSize);
```

Comment:
This threw me for a wild ride... couldn't find the `console.log` or `console.debug` that was spitting out random text in the test log